app/poros/trip.rb

```diff
@@ -0,0 +1,43 @@
+class Trip
+  attr_reader :flight_id,
+             :departure_datetime,
+             :arrival_datetime,
+             :r_departure_datetime,
+             :r_arrival_datetime,
+             :origin_city,
+             :destination_city,
+             :price,
+             :trip_duration,
+             :max_f,
+             :min_f,
+             :max_c,
+             :min_c,
+             :day_feels_like_c,
+             :day_feels_like_f,
+             :sky_coverage,
+             :booking_link,
+             :weather_data_available,
```

Comment:
I think we can safely remove the `weather_data_available` attribute for now.

Reply:
removed, thanks!